server.py

```diff
@@ -1454,6 +1454,27 @@ async def mastered_stuff(ctx: ContextType, *card: str):
     else:
         await ctx.reply(f"The {info.cls_name} {info.name} is NOT mastered.")
 
+@with_savefile("currentmastery")
```

Comment:
don't love this command name, wondering if you have other ideas for it

Reply:
Perhaps "candidates"?

Reply:
I like !candidates quite a lot. Changed